src/number_theory/number_field/embeddings.lean

```diff
@@ -1,179 +1,28 @@
 /-
-Copyright (c) 2021 Ashvni Narayanan. All rights reserved.
+Copyright (c) 2022 Xavier Roblot. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Ashvni Narayanan, Anne Baanen
+Authors: Alex J. Best, Xavier Roblot
 -/
 
-import ring_theory.dedekind_domain.integral_closure
-import algebra.char_p.algebra
+import number_theory.number_field.basic
 import analysis.normed_space.basic
 import topology.algebra.polynomial
```

Comment:
```suggestion
import analysis.normed_space.basic
import number_theory.number_field.basic
import topology.algebra.polynomial
```

Reply:
Thanks!

I am not very confortable with function fields, but if it's only about consolidating things and moving files around, I guess I can do it.